index.js

```diff
@@ -75,7 +75,6 @@ const configureEnv = (env, target) => ({
       target === 'browser' && [
         'emotion',
         {
-          autoLabel: true,
```

Comment:
Weird, `autoLabel: true` is actually what the docs says should be the way to achieve what this PR describes. It looks like this could be a bug: https://github.com/emotion-js/emotion/issues/844

The default for `autoLabel` is `NODE_ENV !== 'production'`, so this change should theoretically make not difference 🤔 🤔 

Either way, if it works like this, then 🤷‍♂ But I think it would be worth adding a couple of tests that assert this behavior to make sure we don't have a regression in the future!